xbmc/interfaces/builtins/AddonBuiltins.cpp

```diff
@@ -22,8 +22,10 @@
 #include "games/tags/GameInfoTag.h"
 #include "guilib/GUIComponent.h"
 #include "guilib/GUIWindowManager.h"
+#include "GUIPassword.h"
```

Comment:
wrong include order
> Included header files have to be sorted (case sensitive) alphabetically [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
thx! missed that one indeed.